internal/common/mqtt/mqttconnection.go

```diff
@@ -115,7 +115,8 @@ func StartMQTTClient(brokerURL string, clientID string) string {
 		return err.Error()
 	}
 	clientConfig.ClientOptions.SetOnConnectHandler(clientConfig.onConnect())
-	URL := clientConfig.SetBrokerURL("tcp")
+	clientConfig.setProtocol()
+	URL := clientConfig.SetBrokerURL()
 	log.Info(logPrefix, " The broker is", URL)
```

Comment:
Before `logPrefix = "[MQTTConnectionMgr]"` should be change to 1logPrefix = "[MQTTConnectionMgr] "`

```suggestion
	log.Info(logPrefix, "The broker is", URL)
```

Reply:
Done!!